README.md

```diff
@@ -77,9 +80,9 @@ Also see:
 
 ```bash
 # Preferred
-$ rake
-
-# Run all the tests without automatically starting jetty.
+$ rake # Run all tests. 
+  
+# Run all the tests including starting/stopping jetty and refreshing all fixtures.
```

Comment:
Okay, I guess that makes this correct.  But I'd like to get rid of that task (in a separate PR). I don't know why you would want to use it that way.